node_modules/browser/web-viewer/actions.js

```diff
@@ -21,15 +22,29 @@ const initial = fromJS({
   icons: {},
   meta: {},
   backgroundColor: null,
-  securityState: 'insecure',
   securityExtendedValidation: false,
   canGoBack: false,
-  canGoForward: false
+  canGoForward: false,
+  theme: theme.defaultTheme
 });
 
-const open = () => initial.merge({id: uuid()});
+// Open new tab
+const open = () => initial.merge({
+  id: uuid()
+});
 exports.open = open;
 
+// Load initial tab.
+// @FIXME there may be a better way to handle this case: a more flexible `open`
+// function perhaps?
+const load = (uri) => initial.merge({
+  id: uuid(),
+  uri: uri,
+  isSelected: true,
+  isFocused: true
+});
+exports.load = load;
```

Comment:
I actually added this because we had a mysterious bug introduced by a hardcoded state object. I feel it's better to keep all the state initializers in one central, searchable place.